include/sdsl/io.hpp

```diff
@@ -384,9 +385,18 @@ uint64_t serialize_vector(const std::vector<T>&		 vec,
 template <typename T>
 void load_vector(std::vector<T>& vec, std::istream& in)
 {
+#if SDSL_HAS_CEREAL
+	{
```

Comment:
Is there a reason why you scope serialization everywhere in this file?

Reply:
To ensure the archive is flushed.

Reply:
But it should be flushed by the end of the function in the next line anyway?

Reply:
Idk, cereal recommends doing it this way since it's the only way to ensure it.

Reply:
> Some archives in cereal can only safely finish flushing their contents upon their destruction.

Whether it is destructed by an additional scope or by the end of the function in the next line does not matter. I don't see any benefit here to add an additional scope.